boards/lilygo/ttgo_t8c3/doc/index.rst

```diff
@@ -0,0 +1,237 @@
+
+   Lilygo ttgo t8-c3
+
+Functional Description
+**********************
+This board is based on the ESP32-C3 with 4MB of flash, WiFi and BLE support. It
+has an USB-C port for programming and debugging, integrated battery charging
+and an on-board antenna. The fitted U.FL external antenna connector can be
+enabled by moving a 0-ohm resistor.
+
+Connections and IOs
+===================
+
+The ttgo t8-c3 board configuration supports the following hardware features:
+
++-----------+------------+------------------+
+| Interface | Controller | Driver/Component |
++===========+============+==================+
+| PMP       | on-chip    | arch/riscv       |
++-----------+------------+------------------+
+| INTMTRX   | on-chip    | intc_esp32c3     |
++-----------+------------+------------------+
+| PINMUX    | on-chip    | pinctrl_esp32    |
++-----------+------------+------------------+
+| USB UART  | on-chip    | serial_esp32_usb |
++-----------+------------+------------------+
+| GPIO      | on-chip    | gpio_esp32       |
++-----------+------------+------------------+
+| UART      | on-chip    | uart_esp32       |
++-----------+------------+------------------+
+| I2C       | on-chip    | i2c_esp32        |
++-----------+------------+------------------+
+| SPI       | on-chip    | spi_esp32_spim   |
++-----------+------------+------------------+
+| TWAI      | on-chip    | can_esp32_twai   |
++-----------+------------+------------------+
+
+.. figure:: img/ttgo_t8c3_pinout.webp
+   :align: center
+   :alt: ttgo t8-c3 pinout
+
+   Lilygo ttgo t8-c3 pinout
+
+
+Start Application Development
+*****************************
+
+Before powering up your Lilygo ttgo t8-c3, please make sure that the board is in good
+condition with no obvious signs of damage.
+
+System requirements
+===================
+
+Prerequisites
+-------------
+
+Espressif HAL requires WiFi and Bluetooth binary blobs in order work. Run the command
+below to retrieve those files.
+
+.. code-block:: console
+
+   west blobs fetch hal_espressif
+
+.. note::
+
+   It is recommended running the command above after :file:`west update`.
+
+Building & Flashing
+*******************
+
+ESP-IDF bootloader
+==================
+
+The board is using the ESP-IDF bootloader as the default 2nd stage bootloader.
+It is build as a subproject at each application build. No further attention
+is expected from the user.
+
+MCUboot bootloader
+==================
+
+User may choose to use MCUboot bootloader instead. In that case the bootloader
+must be build (and flash) at least once.
+
+There are two options to be used when building an application:
+
+1. Sysbuild
+2. Manual build
+
+.. note::
+
+   User can select the MCUboot bootloader by adding the following line
+   to the board default configuration file.
+   ```
+   CONFIG_BOOTLOADER_MCUBOOT=y
+   ```
+
+Sysbuild
+========
+
+The sysbuild makes possible to build and flash all necessary images needed to
+bootstrap the board with the ESP32 SoC.
```

Comment:
Please, use `ESP32-C3`instead of only `ESP32` to refer to this SoC. We also need to update the documentation of other esp32-based boards accordingly. Please, check all entries in this documentation if possible.

Reply:
Hi Sylvio,

I replaces the ESP32 by ESP32-C throughout the documentation.